app/models/bulkrax/csv_entry.rb

```diff
@@ -40,14 +40,6 @@ def self.children_field
       Bulkrax.parent_child_field_mapping[self.to_s] || 'children'
     end
 
-    def keys_without_numbers(keys)
```

Comment:
keys_without_numbers moved to somewhere that both the parser and entry can access it